Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,145 @@
+	<head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        
+		<title>GirlDevelopIt</title>
+		<link href = "css/styles.css" rel = "stylesheet" type ="text/css">
+        
+            <!-- Bootstrap -->
+        <link rel="stylesheet" href="Bootstrap/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="Bootstrap/css/bootstrap.css" />
+        <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+        <!-- Include all compiled plugins (below), or include individual files as needed -->
+        <script src="Bootstrap/js/bootstrap.min.js"></script>
+	</head
+	<body>
+        
+		<div>
+			<img src="https://stackoverflow.blog/wp-content/uploads/2017/08/wordmark-tagline-pink-2.png">
+		</div>
+
+		<!--div about-->
+		<div>
+			<h1>About</h1>
+            
+            <div class = "row">
+
+			<div class = "col-5"> It can be intimidating for women to learn and ask questions when they are in an
+					<strong>extreme minority</strong>. While open and welcoming, today's budding developer
+					community is up to <strong>91% male</strong>. If we can empower more females with the
+					confidence in their technological capabilities <i>we can begin to change this landscape.</i>
+            </div>
+                
+            </div>
+        </div> </br>
+
+		<!--div location-->
+		<div>
+			<h2>Our Locations</h2>
+				<ul>
+					<li>USA
+						<ul>
+								<li>
+									<a href ="https://www.girldevelopit.com/chapters/new-york">New York</a>
+								</li>
+
+								<li>
+									<a href ="https://www.girldevelopit.com/chapters/columbus">Columbus</a>
+								</li>
+
+								<li>
+									<a href ="https://www.girldevelopit.com/chapters/austin">Austin</a>
+								</li>
+
+								<li>
+									<a href ="https://www.girldevelopit.com/chapters/philadelphia">Philly</a>
+								</li>
+							</ul>
+						</li>
+
+						<li>World
+							<ul>
+								<li>
+									<a href = "https://www.girldevelopit.com/chapters/ottawa">Ottawa</a>
+								</li>
+
+								<li>
+									<a href = "https://www.meetup.com/es-ES/topics/girl-develop-it/au/sydney/?country=au&zipstatecity=sydney&state=&radius=25">Sydney</a>
+								</li>
+							</ul>
+						</li>
+					</ul>
+				</div>
+
+				<!--Classes div-->
+				<div>
+					<h2>Upcoming Classes</h2>
+
+					<table>
+						<tr>
+							<th>Location</th>
+							<th>Topic</th>
+						</tr>
+
+						<tr>
+							<td>New York</td>
+							<td>JavaScript & jQuery</td>
+						</tr>
+
+						<tr>
+							<td>San Francisco</td>
+							<td>HTML & CSS</td>
+						</tr>
+					</table>
+                </div> </br>
+        
+                <!--Sign Up div-->
+                <div>
+                    <h2>Sign Up!</h2>
+                    
+                    <label>
```

Comment:
Wrap all the form inputs into a `<form>` tag and also check bootstrap's forms: https://getbootstrap.com/docs/4.1/components/forms/